submissions/elijahZgambo/Database/init.sql

```diff
@@ -0,0 +1,53 @@
+/* Creating food_bag database */
+create database food_bag;
+show databases;
+use food_bag;
+/* Creating food table */
+create table food ( 
+	id int not null auto_increment, 
+	type ENUM('fast_food', 'vegetable_fruits', 'drinks_cocktails', 'restaurants') not null default 'fast_food',
+    primary key (id) 
+    );
+
+/* Creating admins table */
+create table  admins (
+	id int not null auto_increment, 
+	name varchar(250) not null, 
+    type int not null, 
```

Comment:
The admins' table will contain the system admins of the food bag platform and the vendors. They both will be differentiated by type. The system admins can have a type of 0 and vendors a type of 1 when they are saved for differentiation. So change this to Boolean instead of INT.